src/pages/product-details/comments/comments-item/reply-form/reply-form.js

```diff
@@ -72,7 +71,7 @@ const ReplyForm = ({ cancel, commentId, refetchComments }) => {
           onChange={handleCommentChange}
           onBlur={handleBlur}
           error={!!errors.text}
-          helperText={errors.text || ''}
+          helperText={errors.text ? t('error.replyCommentField') : ''}
```

Comment:
You don't need to use ternary operator when else statement returns nothing. Just use '&&' operator instead which is better approach in this case